lib/efi_event.h

```diff
@@ -147,7 +152,7 @@ typedef struct {
     UINT64 StartingLBA;
     UINT64 EndingLBA;
     UINT64 Attributes;
-    BYTE PartitionName[72];
+    UTF16_CHAR PartitionName[36];
```

Comment:
@josephlr, what is the magic number 36?

Reply:
That's just the definition in the EFI spec:
```c
typedef struct {
    EFI_GUID PartitionTypeGUID;
    EFI_GUID UniquePartitionGUID;
    EFI_LBA StartingLBA;
    EFI_LBA EndingLBA;
    UINT64 Attributes;
    CHAR16 PartitionName[36];
} PACKED EFI_PARTITION_ENTRY;
```

See section `5.3.3` of the [UEFI Spec v2.9](https://uefi.org/sites/default/files/resources/UEFI_Spec_2_9_2021_03_18.pdf).

The GPT format chose `36` so that `sizeof(EFI_PARTITION_ENTRY) == 128` ([godbolt link](https://godbolt.org/z/4hxbsYja6)) so that the entry fits exactly in two x86 cache lines.

Reply:
@idesai, do you want me to add a comment explaining this, or is that implied from the structure of this file?

Reply:
@josephlr yes please.

Reply:
Done